qa/standalone/misc/ok-to-stop.sh

```diff
@@ -264,6 +264,8 @@ function TEST_0_osd() {
     ceph osd ok-to-stop 3 || return 1
     ! ceph osd ok-to-stop 0 1 || return 1
     ! ceph osd ok-to-stop 2 3 || return 1
+    ceph osd ok-to-stop 0 --max 2 | grep '[0]' || return 1
+    ceph osd ok-to-stop 1 --max 2 | grep '[1]' || return 1
```

Comment:
might want to update `doc/man/8/ceph.rst` accordingly. to me, the meaning of `--max` is obvious at the first glance. would be ideal if we can accompany it with some example in document or man page.